src/components/Widgets/Markdown/serializers/remarkEscapeMarkdownEntities.js

```diff
@@ -248,6 +248,12 @@ function escape(delim) {
  */
 export default function remarkEscapeMarkdownEntities() {
   const transform = (node, index) => {
+    /**
+     * Shortcode nodes will intentionally inject markdown entities in text node
+     * children not be escaped.
```

Comment:
Is there a typo here, or do I just not understand what it means?

Reply:
Nope, that's a typo.